app/services/back_end_service.rb

```diff
@@ -28,6 +28,15 @@ def create_user(user_params)
       )
     end
 
+    def update_user(user_params, user_id)
```

Comment:
Did we confirm that this works?  (i.e. running two servers)?   Took a few tries to get this working on POST requests